pom.xml

```diff
@@ -80,6 +80,37 @@
         <useReleaseProfile>true</useReleaseProfile>
     </properties>
 
+    <repositories>
+        <repository>
+            <id>central</id>
+            <name>Central</name>
+            <url>https://repo.maven.apache.org/maven2/</url>
+            <releases>
+                <enabled>true</enabled>
+                <updatePolicy>never</updatePolicy>
+            </releases>
+            <snapshots>
+                <enabled>false</enabled>
+                <updatePolicy>never</updatePolicy>
+            </snapshots>
+        </repository>
+
+        <repository>
+            <id>kogito-staging-repository-group</id>
+            <name>Kogito Staging Repositories</name>
+            <url>https://repository.jboss.org/nexus/content/groups/kogito-public/</url>
+            <layout>default</layout>
+              <releases>
+                <enabled>true</enabled>
+                <updatePolicy>never</updatePolicy>
+              </releases>
+              <snapshots>
+                <enabled>false</enabled>
+                <updatePolicy>never</updatePolicy>
+              </snapshots>
+        </repository>
+    </repositories>
```

Comment:
@evacchi A good trick to avoid this is add these in ~/.m2/settings.xml instead with a profile:

```
<?xml version="1.0" encoding="UTF-8"?>
<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
  xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0 http://maven.apache.org/xsd/settings-1.0.0.xsd">
    <profile>
      <id>kogito-staging</id>
      <repositories>
        <repository>
            <id>kogito-staging-repository-group</id>
            <name>Kogito Staging Repositories</name>
            <url>https://repository.jboss.org/nexus/content/groups/kogito-public/</url>
            <layout>default</layout>
              <releases>
                <enabled>true</enabled>
                <updatePolicy>never</updatePolicy>
              </releases>
              <snapshots>
                <enabled>false</enabled>
                <updatePolicy>never</updatePolicy>
              </snapshots>
        </repository>
      </repositories>
    </profile>
  </profiles>
</settings>
```

and then run `mvn clean install -Pkogito-staging`

Reply:
yes but I can't add `~/.m2/settings.xml` on the CI env :)